### ISSUE TITLE
[debops.owncloud] error_page support for Nextcloud 18+

### ISSUE THREAD

**Issue by jankowa** [debops/debops#1492]
According to https://github.com/nextcloud/text/issues/820 the 

```
error_page 403 /core/templates/403.php;
error_page 404 /core/templates/404.php;
```

in nginx config is deprecated.

If the lines are present there is this annoying workspace circle moving forever on the top, as long as there is no README.md inside the directory.

The workspace error appeared with nextcloud >= 18.

**Comment by jankowa** [comment]
This should fix it:
```
diff --git a/ansible/roles/owncloud/defaults/main.yml b/ansible/roles/owncloud/defaults/main.yml
index 8c016659b..895f5ea73 100644
--- a/ansible/roles/owncloud/defaults/main.yml
+++ b/ansible/roles/owncloud/defaults/main.yml
@@ -2164,9 +2164,13 @@ owncloud__nginx__dependent_servers:
       # Uncomment if your server is build with the ngx_pagespeed module
       # This module is currently not supported.
       #pagespeed off;
-
+      #
+      {% if not (owncloud__variant == "nextcloud" and
+                     owncloud__release is version("18.0", ">")) %}
       error_page            403             /core/templates/403.php;
       error_page            404             /core/templates/404.php;
+      {% endif %}
 
 
       # Avoid to send the security headers twice as ownCloud
```

**Comment by ypid** [comment]
Thanks for the patch, looks good to me. I guess I will have to update my instances at some point and will test this. But not in the next weeks.

**Comment by jankowa** [comment]
Thanks for having a look. If I come over it, I'll supply a pull request.

one little change: It has to be: 
`                       owncloud__release is version("18.0", ">=")) %}`

**Comment by ypid** [comment]
Addressed in: #1524
